### ISSUE TITLE
On a phone, the mobile menu is a toggle

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2502]
## Source

Pacific ADA

## Describe the bug

On a phone, the mobile menu is a toggle.  Even when it is closed and not expanded, a screen reader user can navigate through the elements in the menu.  These elements should be hidden from screen reader view until the menu is selected.  See video.

## Screenshots

https://github.com/proudcity/wp-proudcity/assets/1569463/522c8968-9a94-4b7f-b24f-fc3023f63129
